types/src/v2/error.rs

```diff
@@ -24,6 +24,27 @@ impl<'a> fmt::Display for JsonRpcError<'a> {
 	}
 }
 
+/// JSON-RPC error object.
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
+#[serde(deny_unknown_fields)]
+pub struct JsonRpcErrorObjectOwned {
```

Comment:
I think this should be named just `RpcError`. It being owned is an implementation detail imo.